src/test/java/com/marginallyclever/makelangelo/MissingTranslationTest.java

```diff
@@ -65,7 +65,7 @@ public void verifyThatMatcherWorks_results() throws IOException {
         // matches.txt contains few entries with translation and one without. The list should not be empty
         searchInAFile(new File("src/test/resources/translator/matches.txt"), results);
 
-        assertEquals(List.of("src/test/resources/translator/matches.txt:4: unknownKey"), results);
+        assertEquals(List.of("src/test/resources/translator/matches.txt:4: unknownKey".replaceAll("/", File.separator)), results);
```

Comment:
Sorry i may have miss something
now we have
~~~
Error:  Errors: 
Error:    MissingTranslationTest.verifyThatMatcherWorks_results:68 � IllegalArgument cha...
~~~
and I'm confused as to why. encoding problem of my message if you copy pasted it?

Reply:
I changed the output of the test, on IntelliJ, I can now click in the log on the file and it opens directty to the file at the line. Also update the assertion to not fails on windows (hopefully)

![image](https://user-images.githubusercontent.com/23615562/151840550-289ffd0f-d5b8-4588-a17c-580d7de7fd33.png)